CMakeLists.txt

```diff
@@ -39,14 +34,25 @@ if (VUL_WERROR)
     endif()
 endif()
 
-find_package(PythonInterp 3.7.2 REQUIRED)
+add_library(VulkanLayerUtils STATIC)
```

Comment:
This can be fixed with target_sources and source_groups. I'll add it.

Reply:
Actually given we are going to have multiple libraries it doesn't make sense to have it in the top level CMakeLists.txt

If we only had 1 however it would be standard practice to have it in the top level CMakeLists.txt however.